setup.py

```diff
@@ -13,7 +13,7 @@
 
 REQUIREMENTS_TEST = open(os.path.join(HERE, "requirements_test.txt")).readlines()
 REQUIREMENTS = [
-    "smart_open[http]",
+    "smart_open[s3,http]@git+https://github.com/ddelange/smart_open@patch-1",
```

Comment:
ref https://github.com/RaRe-Technologies/smart_open/pull/702

Reply:
when I start a fresh pod running `stevearc/pypicloud:1.3.6`, k8s still registers `141Mi`. potentially a memprofiling candidate 🤔

Reply:
memprofiling using [`filprofiler`](https://pypi.org/project/filprofiler/)

a default `server.ini` generated using local filesystem option, killed once the server booted succesfully:
```
fil-profile run -m pyramid.scripts.pserve server.ini
```
https://user-images.githubusercontent.com/14880945/178993376-ef2c9111-33fa-4dda-b27e-956fcbe1ccb0.svg